docs/api_walkthrough.rst

```diff
@@ -160,33 +160,41 @@ With the payload::
+This will automatically connect to and open channels with three random peers in the token network, with 20% of the funds deposited to each channel. Furthermore it will leave 40% of the funds initially unassigned. This will allow new nodes joining the network to open bi-directionally funded payment channels with this node in the same way that it just opened channels with random nodes already in the network. The default behaviour of opening three channels and leaving 40% of the tokens for new nodes to connect with, can be changed by adding ``"initial_channel_target": 3`` and ``"joinable_funds_target": 0.4`` to the payload and adjusting the default value.
 
 The user node is now connected to the token network for the AET token, and should have a path to all other nodes that have joined this token network, so that it can transfer tokens to all nodes participating in this network. See the :ref:`Transferring tokens <transferring-tokens>` section for instructions on how to transfer tokens to other nodes.
 
 
 .. _leave-network:
 
 Leave
------
+------
 If at some point it is desired to leave the token network, the ``leave`` endpoint is available. This endpoint will take care of closing and settling all open channels for a specific in the token network::
 
     DELETE /api/v1/connection/0xc9d55C7bbd80C0c2AEd865e9CA13D015096ce671
 
 This call will take some time to finalize, due to the nature of the way that settlement of payment channels work. For instance there is a ``settlement_timeout`` period after calling ``close`` that needs to expire before ``settle`` can be called.
 
+For reasons of speed and financial efficiency the ``leave`` call will only close and settle channels for which your node has received a transfer.
+
+If you want to override the default behaviour and leave all open channels add the follow payload::
+
+  {
+      "only_receiving_channels": false
+  }
+
 
 .. _transferring-tokens:
 
 Transferring tokens
-===================
-So far it has been shown how to bootstrap a token network, how to join an already existing token network, and how to leave a token network. However, it still remains to take a look at what Raiden is really all about - transferring tokens from one node to another in off-chain payment channels. It's assumed that a node is connected to the token network of the AET token mentioned above. In this case the node is connected to five peers, since the standard ``connect()`` parameters were used.
+====================
+So far it has been shown how to bootstrap a token network, how to join an already existing token network, and how to leave a token network. Next we see how to transfer tokens from one node to another in off-chain payment channels. For the example it's assumed that a node is connected to the token network of the AET token mentioned above. In this case the node is connected to five peers, since the standard ``connect()`` parameters were used.
```

Comment:
Keep it neutral please. So don't use `we`. Maybe write "Next it is shown how to transfer tokens...".